web/styles/zulip.css

```diff
@@ -115,6 +115,16 @@ body {
        We expect `resize.js` to update this once UI is initialized. */
     --recent-topics-filters-height: 50px;
 
+    /* Overlay heights for streams modal */
+    --modal-container-height: 95vh;
+    --modal-container-max-height: 1000px;
+    /* .subscriptions-header */
+    --modal-container-subscriptions-header-height: 45px;
+    /* .display-type */
+    --modal-container-subscriptions-display-type-height: 44px;
+    /* .modal-footer */
+    --modal-container-subscriptions-modal-footer-height: 60px;
```

Comment:
I guess I'd be hoping for a world where we'd also change the CSS for the modal to use this variable, so that things are actually coupled, but we can do that sort of cleanup in follow-ups; this is already a lot more readable.